samples/bluetooth/periodic_sync_rsp/src/main.c

```diff
@@ -229,7 +229,7 @@ int main(void)
 		err = bt_le_adv_start(
 			BT_LE_ADV_PARAM(BT_LE_ADV_OPT_ONE_TIME | BT_LE_ADV_OPT_CONNECTABLE,
 					BT_GAP_ADV_FAST_INT_MIN_2, BT_GAP_ADV_FAST_INT_MAX_2, NULL),
-			NULL, 0, sd, ARRAY_SIZE(sd));
+			ad, ARRAY_SIZE(ad), NULL, 0);
```

Comment:
So what you are doing here is using advertising data instead of scan response data for the name. 

That's fine, but either should work, assuming that the remote device are using active scan. 

I don't think it's right to call it broken, as this should still work, so I would suggest to reword the commit message.

Reply:
The alternative is to modify `periodic_adv_rsp` to use active scan instead of `BT_LE_SCAN_PASSIVE_CONTINUOUS`